client/wfnews-war/src/main/angular/src/app/components/report-of-fire/review-page/rof-review-page.component.ts

```diff
@@ -56,15 +56,15 @@ export class RoFReviewPage extends RoFPage implements AfterViewInit{
       case 'fire-size-page' :
         return this.reportOfFire.fireSize.toString();;
       case 'response-details-page' :
-        return this.reportOfFire.signsOfResponse.toString();
+        return this.reportOfFire.ifSignsOfResponse.charAt(0).toUpperCase() + this.reportOfFire.ifSignsOfResponse.slice(1);
```

Comment:
This kind of statement would benefit from a null check, and a comment explaining why you're doing this.